FakeServer/Program.cs

```diff
@@ -122,15 +123,31 @@ private static int Run(string[] args, Dictionary<string, string> initialData)
                     app.RemainingArguments.Add(optionsUrls.Value());
                 }
 
+                if (optionInit.HasValue()) {
+                    app.RemainingArguments.Add("--init");
```

Comment:
RemainingArguments are passed to server startup. App should exit after init so these are no needed.